Makefile

```diff
@@ -35,7 +35,7 @@ $(VENV)/bin/activate: sdk/python/setup.py
 	@echo "VENV=$(VENV)"
 	@test -d $(VENV) || python3 -m venv $(VENV)
 	@$(VENV)/bin/pip show kfp-tekton >/dev/null 2>&1 || $(VENV)/bin/pip install -e sdk/python
-	@touch $(VENV)/bin/activate
+#   @touch $(VENV)/bin/activate
```

Comment:
we need to figure out a way to keep the `touch` command when not running as a GitHub action to maintain full functionality of automatically updating the virtual environment when there was a change to `setup.py`

Reply:
let me add an env to switch the behavior between different CI then.

Reply:
ok I added a condition to touch the file if not specify for github action.

Here is the sample github action run
https://github.com/Tomcli/kfp-tekton/actions/runs/950503294

Reply:
it looks like the the `VIRTUAL_ENV` gets (re)created for the Makefile targets that require it. We may want to find a way to re-use the same `VIRTUAL_ENV` for all the Make targets. But we can optimize this in the future.